### PR TITLE
Add bacon.js family to registry

### PR DESCRIPTION
**Pull request by ledmonster**
Added shortcuts for bacon.js related libraries.

bacon.matchers uses baconjs 0.7.6 by default, so overrided it to use an installed (latest) bacon.js.

### CODE REVIEW

File: package-overrides/npm/bacon-matchers@0.4.0.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "baconjs": "*"
```
Comment (guybedford) [inline package-overrides/npm/bacon-matchers@0.4.0.json]: Perhaps make this `^0.7.0`?

**Comment by guybedford** [comment]
Thanks for the PR, let me know what you think about using the semver range for the override.

**Comment by ledmonster** [comment]
@guybedford, thank you for your comment. I may be misunderstanding the spec of jspm, so let me explain my concern.

When I run
```
jspm install baconjs
jspm install "bacon-matchers=npm:bacon.matchers"
```
jspm generates following config.js
```
...
System.config({
  "map": {
    "bacon-matchers": "npm:bacon.matchers@0.4.0",
    "baconjs": "npm:baconjs@0.7.53",
...
    "npm:bacon.matchers@0.4.0": {
      "baconjs": "npm:baconjs@0.7.6"
    },
    "npm:baconjs@0.7.53": {
      "assert": "github:jspm/nodelibs-assert@0.1.0",
      "buffer": "github:jspm/nodelibs-buffer@0.1.0",
      "fs": "github:jspm/nodelibs-fs@0.1.2",
      "path": "github:jspm/nodelibs-path@0.1.0",
      "process": "github:jspm/nodelibs-process@0.1.1"
    },
    "npm:baconjs@0.7.6": {
      "buffer": "github:jspm/nodelibs-buffer@0.1.0",
      "process": "github:jspm/nodelibs-process@0.1.1"
    },
```

But I want bacon.matchers to use same version as `"baconjs": "npm:baconjs@0.7.53",`
```
    "npm:bacon.matchers@0.4.0": {
-      "baconjs": "npm:baconjs@0.7.6"
+      "baconjs": "npm:baconjs@0.7.53"
    },
```
bacon.matchers is a baconjs plugin, and it adds extra methods on Bacon object.

You can see my simple trial here:

- https://github.com/ledmonster/jspm-trial

**Comment by guybedford** [comment]
@ledmonster thanks, yes I did try it out as well.

If you change the override for bacon.matchers to:

```json
{
  bacon: "^0.7.0"
}
```

This will still work as you want it to, but then be properly ranged to not break in future as well.

**Comment by ledmonster** [comment]
@guybedford thank you, I understand.

I changed the override and rebased to current master.

**Comment by guybedford** [comment]
Perfect, thanks.

**Pull request merged**